src/main.rs

```diff
@@ -68,21 +87,18 @@ fn main() {
                                       .arg_from_usage("<cmd>... 'cargo command to run'"))
                       .get_matches();
 
-    let mut cargo_cmd = Command::new(CARGO);
-    let mut banner = String::from("Executing ") + CARGO;
+    let commands = matches.subcommand_matches("multi")
+                          .and_then(|m| m.values_of("cmd"))
+                          .expect("No cargo commands provided")
+                          .map(|arg| arg.to_string())
+                          .collect::<Vec<_>>();
 
-    if let Some(arg_cmd) = matches.subcommand_matches("multi")
-                                  .and_then(|m| m.values_of("cmd")) {
-        for arg in arg_cmd {
-            cargo_cmd.arg(arg);
-            banner = banner + " " + arg;
-        }
-    }
+    let banner = format!("Executing {} {}", CARGO, commands.join(" "));
 
     announce(&banner);
```

Comment:
Won't this be wrong?  It won't have `--manifest-file` in it unlike the real deal.